content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: CPUManager goes GA'
+date: 2022-12-01
+slug: cpumanager-goes-GA
+---
+
+**Author:**
+Francesco Romani (Red Hat)
+
+The CPU Manager is a part of the kubelet, the Kubrnetes node agent, which enables the user to allocate exclusive CPUs to containers.
+Since its [Beta graduation in 2018](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager adoption steadily grown and become a staple
```

Comment:
```suggestion
Since its [Beta graduation in 2018](blog/2018/07/24/feature-highlight-cpu-manager/), the CPU Manager adoption has steadily grown making it a staple component
```